tests/wasmfs/wasmfs_seek.c

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2021 The Emscripten Authors.  All rights reserved.
+ * Emscripten is available under two separate licenses, the MIT license and the
+ * University of Illinois/NCSA Open Source License.  Both these licenses can be
+ * found in the LICENSE file.
+ */
+
+#include <assert.h>
+#include <errno.h>
+#include <fcntl.h>
+#include <stdio.h>
+#include <unistd.h>
+
+// FIXME: The existing io.c file implements devices, which haven't been
+// implemented in the new FS yet.
+// This test simply implements the seek portion.
+
+/*
+ * Copyright 2021 The Emscripten Authors.  All rights reserved.
+ * Emscripten is available under two separate licenses, the MIT license and
+ * the University of Illinois/NCSA Open Source License.  Both these licenses
+ * can be found in the LICENSE file.
+ */
```

Comment:
seems odd to have two copyright notices in one file - copy-paste?

Reply:
Oh yeah I tried to merge in the test from `io.c` here